shaders/src/getters.fs

```diff
@@ -108,3 +108,31 @@ bool isDoubleSided() {
     return materialParams._doubleSided;
 }
 #endif
+
+/**
+ * Returns the cascade index for this fragment (between 0 and CONFIG_MAX_SHADOW_CASCADES - 1).
+ */
+uint getShadowCascade() {
```

Comment:
shouldn't this be inside the `defined(HAS_SHADOWING)`?

Reply:
`getShadowCascade` is also used for visualizing the different cascades. For this to work correctly, all fragments (even those not affected by shadowing) should be colored with their respective cascade.